<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SongPage</class>
 <widget class="QDialog" name="SongPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_pageTitle">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adelle Sans Devanagari</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Song Creation Page</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>231</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Add New Song</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>20</y>
      <width>61</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_artist">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Artist:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_album">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Album:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_duration">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Duration:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_title"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_artist"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_album"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_duration"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_addToSongLibrary">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Add to Song Library</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>271</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Song Library</string>
   </property>
   <widget class="QTableWidget" name="tableWidget_songs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_displaySongs">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Display Songs</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
